packages/dataviews/src/field-types/integer.tsx

```diff
@@ -4,15 +4,17 @@
 import type { SortDirection } from '../types';
 
 function sort< Item >(
-	itemA: Item,
-	itemB: Item,
-	direction: SortDirection,
-	getValue: ( args: { item: Item } ) => any
+	a: {
+		item: Item;
```

Comment:
Why have Item here at all? I was thinking `a: any, b: any` should be sufficient no?

Reply:
@youknowriad please be welcome to push. In my mind both functions are instances of the same idea, so they'd have the same signature.

Reply:
For me field types and fields work at different levels, field types work on values, fields work on items, so it doesn't make sense to have the same API.

I've pushed what I had in mind in [621ac69](https://github.com/WordPress/gutenberg/pull/64064/commits/621ac69ad314178c8d6cabfc723a25b3b05f78c0) 

Feel free to revert if you don't like it.

Reply:
[621ac69](https://github.com/WordPress/gutenberg/pull/64064/commits/621ac69ad314178c8d6cabfc723a25b3b05f78c0) works for me.